content/tutorials/server/smtp-servers/set-up-oauth-microsoft-365/index.md

```diff
@@ -7,10 +7,82 @@ previous-topic: smtp-servers
 next-topic: smtp-servers
 ---
 
-# EHow to implement OAUTH using DNN and Microsoft 365
+# How to implement OAUTH using DNN and Microsoft 365
 
-## Placeholder
+## Introduction
+This page describes the nessecery steps to make OAUTH SMTP work with Microsoft 365.   
 
-This is a new page. Tutorial wil follow soon
+## Setup steps
+1. Check some settings in DNN
+2. Create a mailbox in M365 Exchange 
+3. Create an APP registration in the M365 Azure
+4. Grant access to the app to use the mailbox
+5. Setup the mailbox in DNN 
 
+## 1. Check some settings in DNN
+> [!WARNING]
+> To avoid errors with the authentication in step 5, make sure that the SSL settings are correct. It must be set on **ON**. 
+* Personabar - Security - More - SSL Settings: Make sure that SLL Settings = **ON** (/images/scr-m365-oauth-sslsetting.png)  
+* Personabar - Security - Login settings: Check the site administrator account. You need to use this e-mailaddress in M365, because the site administrator is the sender.

```

Comment:
```suggestion
* Personabar - Security - Login settings: Check the site administrator account. You need to use this e-mail address in M365, because the site administrator is the sender.
```